src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -981,9 +982,11 @@ public function timestampTz($column, $precision = 0)
      */
     public function timestamps($precision = 0)
     {
-        $this->timestamp('created_at', $precision)->nullable();
+        $this->timestamp('created_at', $precision)->useCurrent();
 
-        $this->timestamp('updated_at', $precision)->nullable();
+        $this->timestamp('updated_at', $precision)->default(
+            new Expression('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP')
```

Comment:
`CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP` is MySQL-specific feature.

Reply:
Thank you, I didn't think about it.